src/MCPClient/tests/test_sanitize.py

```diff
@@ -203,10 +216,19 @@ def test_sanitize_object_names(self):
             assert File.objects.get(
                 currentlocation="%transferDirectory%objects/takusan_directories/need_sanitization/checking_here/evelyn_s_photo.jpg"
             )
-            assert Event.objects.filter(
+            event = Event.objects.filter(
                 file_uuid="47813453-6872-442b-9d65-6515be3c5aa1",
                 event_type="name cleanup",
-            ).exists()
+            )
+
+            # Test the event identifier for name cleanup was written for our object.
+            assert event is not None, "Event not returned from database"
```

Comment:
Above you're doing:

```
event = Event.objects.filter(
    file_uuid="47813453-6872-442b-9d65-6515be3c5aa1",
    event_type="name cleanup",
)
```

I think that it's preferably to use `get` in L219.
The assertion in L225 becomes unnecessary.